js/index.js

```diff
@@ -82,11 +80,12 @@ function passwordScore(pass) {
     }
     return {score: score, error: error};
 }
-$('.signup-form').find('#password').keyup(function(ele) {
-	var scoreData = passwordScore($(this).val()),
+
+function passwordFieldKeyUp(t) {
```

Comment:
@fatimarafiqui Let's add block comments to this file for methods that would explain what the method does.